crocket/src/main/java/com/crocket/DrawPanel.java

```diff
@@ -0,0 +1,24 @@
+package com.crocket;
+import java.awt.*;
+import javax.swing.ImageIcon;
+import javax.swing.JLabel;
+
+
+public class DrawPanel extends JLabel{
+    
+    //temp placement
```

Comment:
What do you mean with this comment?

Reply:
I mean that when we have a proper model, those lines will be moved. I just needed them to get the view working